src/Account.ts

```diff
@@ -181,7 +181,7 @@ export class Account {
         let response: Response;
         let retryCount = 0;
         init.headers["user-agent"] = Constants.User_Agent;
-        init.headers["x-user-agent"] = Constants.X_User_Agent[this.region];
+        init.headers["x-user-agent"] = Constants.X_User_Agent(this.region);
```

Comment:
```suggestion
        init.headers["x-user-agent"] = Constants.X_User_Agent[this.region];
```

Reply:
Can you explain how you would like this to work? I've changed the X_User_Agent to be a function instead of a dictionary, because it takes in the region and brand to construct the user agent string. For account it uses the default (BMW).
Before this change the user agent was only different for each region. However, the brand in the user agent is what ConnectedDrive uses to determine to fetch BMW or Mini vehicles.

Reply:
Sorry my bad, however, I do not see you passing the brand here, which means it will always default to BMW for the ```x-user-agent``` header.

Reply:
Yes, that's correct. For the authentication it doesn't matter which brand is used, so we use the default here (BMW). For the other operations on the vehicles it does matter though. I've basically just copied the approach that's used in bimmer_connected [here](https://github.com/bimmerconnected/bimmer_connected/blob/77b4d3f8361f0cf64ffe046bf71e30eb4126dc91/bimmer_connected/api/authentication.py#L375-L376).

Reply:
If you tested it on a Mini, I am fine with resolving this. As I could not test, hence, I did not implement this. However, for the sake of consistency better to also add it here. Will add this hopefully in the next update.